htlcswitch/switch.go

```diff
@@ -723,7 +723,7 @@ func (s *Switch) htlcForwarder() {
 
 			// Otherwise, we'll log this diff, then accumulate the
 			// new stats into the running total.
-			log.Infof("Sent %v satoshis received %v satoshi "+
+			log.Infof("Sent %v received %v "+
```

Comment:
I'd prefer to print satoshis instead (as was originally attempted!). We can do this easily by casting `diffSatSent`, and `diffSatRecv` to an `int64` before passing them as the arguments into the logger's `Infof` method.